src/edu/stanford/nlp/ie/machinereading/domains/ace/AceReader.java

```diff
@@ -436,25 +436,25 @@ private EntityMention convertAceEntityMention(AceEntityMention entityMention, St
     int extEnd = ext.getTokenEnd() - tokenOffset + 1;
     if (extStart < 0) {
       logger.severe("READER ERROR: Invalid extent start " + extStart + " for entity mention " + entityMention.getId() + " in document " + docId + " in sentence " + sentence);
-      logger.severe("This may happen due to incorrect EOS detection. Adjusting entity extent.");
+      logger.finest("This may happen due to incorrect EOS detection. Adjusting entity extent.");
```

Comment:
Maybe `incorrect` should be in keyword set.

Reply:
Yes please add it.

Reply:
Oh, actually, I think this one is good because this is about the error *details* rather than the error.